modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -700,16 +700,15 @@ class CV_EXPORTS_W QRCodeDetector
      @param points Quadrangle vertices found by detect() method (or some other algorithm).
      @param straight_qrcode The optional output image containing rectified and binarized QR code
      */
-    CV_WRAP cv::String decode(InputArray img, InputArray points, OutputArray straight_qrcode = noArray());
 
+    CV_WRAP std::vector<std::string> decode(InputArray img, InputArrayOfArrays points, OutputArrayOfArrays straight_qrcode);
```

Comment:
Do we have python test for QR codes? Need to check that `std::vector<std::string>` works fine. Otherwise need to replace to `std::vector<cv::String>`